src/lucky_record/query_builder.cr

```diff
@@ -103,6 +103,15 @@ class LuckyRecord::QueryBuilder
     self
   end
 
+  def where
+    self
+  end
+
+  def not(where_clause : LuckyRecord::Where::SqlClause)
```

Comment:
This isn't needed as well strictly speaking. Since `Queryable` only uses `Equal` we could get by without having `not` and `where` on the `QueryBuilder`. Am I missing where all the other `SqlClause`es are being used?